src/core/security/credentials.c

```diff
@@ -179,8 +179,8 @@ void grpc_server_credentials_set_auth_metadata_processor(
   GRPC_API_TRACE(
       "grpc_server_credentials_set_auth_metadata_processor("
       "creds=%p, "
-      "processor=grpc_auth_metadata_processor { process: %lx, state: %p })",
-      3, (creds, (unsigned long)processor.process, processor.state));
```

Comment:
So, we've already done BadThings right now in the code, by having `%ll` printf-modifiers sprinkled all around, used as 64-bits printer. Technically however, there's no proper cross-platform, printf-modifier. We could define something inttypes.h style, which a PRIu64 macro, basically.